package.json

```diff
@@ -49,12 +51,16 @@
     "grunt-concurrent": "1.0.0",
     "grunt-contrib-jshint": "0.10.0",
     "grunt-fh-build": "0.5.0",
+    "grunt-jsxgettext": "^1.0.0",
     "grunt-open": "0.2.3",
     "grunt-plato": "1.0.0",
     "grunt-shell": "0.6.4",
     "istanbul": "0.2.7",
+    "jsxgettext": "^0.8.2",
     "load-grunt-tasks": "0.4.0",
     "nock": "5.0.0",
+    "node-gettext": "^1.0.1",
+    "os-locale": "^1.4.0",
```

Comment:
These dependencies, `node-gettext` and `os-locale` already exist in `dependencies` so I don't think they should be needed here in `devDependencies` (or maybe *just* in `devDependencies` if not used at run time). Also, if they're going to be in `dependencies`, they'll need to be added to the shrinkwrap file to make sure they actually get installed on the user's machine.

The most consistent way we've got currently of shrinkwrapping would go something like this:

```
rm -rf node-modules
npm i --production
npm i node-gettext@1.0.1 --save
npm i os-locale@1.4.0 --save
npm cache clean
npm shrinkwrap
```

Also, notice that for our other dependencies, we've removed the version "unspecifiers" (e.g. ^, ~, >, *, etc). Could you change these new dependencies to match?